src/win/util.c

```diff
@@ -1778,3 +1778,20 @@ int uv_os_uname(uv_utsname_t* buffer) {
   buffer->machine[0] = '\0';
   return r;
 }
+
+int uv_gettimeofday(uv_timeval_t* tv) {
+  /* Based on https://doxygen.postgresql.org/gettimeofday_8c_source.html */
+  const uint64_t epoch = (uint64_t) 116444736000000000ULL;
```

Comment:
BTW, Windows SDK's as far back as VS9 have `sys/time.h` so this can be unified code.
(including `timezone` and `get_timezone`)

Reply:
@refack are you saying that `gettimeofday()` is available on all versions of Windows that we support? If that's the case, we don't need anything to live in libuv 😄

Reply:
No. I believe [`gettimeofday`](https://pubs.opengroup.org/onlinepubs/009696799/functions/gettimeofday.html) is posix

But we have full [C-runtime](https://en.cppreference.com/w/c/chrono) so you can do:
```c
    time_t t;
    struct tm buf;
	t = time(NULL)
    gmtime_s(&t, &buf));
```

Reply:
gmtime_s() is a C11-ism so that probably means you can't use it with VS 2008 or 2010 (which we still support because of python 2) and plain gmtime() isn't thread-safe.